TrafficCapture/dockerSolution/src/main/docker/migrationConsole/setupIntegTests.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Example usage: ./setupIntegTests.sh https://github.com/opensearch-project/opensearch-migrations.git main
+
+git_http_url=$1
+branch=$2
```

Comment:
Can you update the usage of these variables within this file to be double quoted please?

e.g on line 14:  `git remote add -f origin "$git_http_url"`

Reply:
Have updated now